File: zero/Maruko.Zero/Internal/Service/Menu/DTO/RoleMenuResponse.cs
using System.Collections.Generic;

namespace Maruko.Zero
{
    public class RoleMenuResponse
    {
        public List<string> MenuIds { get; set; } = new List<string>();
        public List<MenuModel> List { get; set; } = new List<MenuModel>();
    }

    public class MenuModel
    {
        public string Id { get; set; }
        public string Lable { get; set; }
        public List<MenuModel> Children { get; set; } = new List<MenuModel>();
    }
}

File: act/act-modules/act-modules-zero/src/main/java/com/act/modules/zero/application/services/menu/dto/RoleMenuResponse.java
package com.act.modules.zero.application.services.menu.dto;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

@Data
public class RoleMenuResponse {
    private List<String> menuIds;
    private List<MenuModel> list;

    public RoleMenuResponse() {
        menuIds = new ArrayList<>();
        list = new ArrayList<>();
    }
}
